common/web/types/src/schemas.ts

```diff
@@ -0,0 +1,30 @@
+
+// @ts-expect-error(2821) import assertions
+import kpj from './schemas/kpj.schema.json' assert { type: 'json' };
```

Comment:
Huh, that's a different pattern than I'd seen mentioned elsewhere for importing from JSON.

I'm used to the suggestions I see at this StackOverflow post: https://stackoverflow.com/q/49996456.

In particular: https://www.typescriptlang.org/tsconfig#resolveJsonModule

I could see using import assertions if we wanted the schemas to dynamically update based on hosted endpoint of one of our sites - that seems to be the intended purpose for the feature that I've noted.  (Cross-origin JSON is safe 'cause it's not JS; the assertion prevents CORS issues there.)
- Based on what I'm seeing mentioned about them here, admittedly: https://v8.dev/features/import-assertions

Is there a different reason we're using import assertions here?

Reply:
Good question! I am using import assertions because Node 18 requires it (since Node 17.5 apparently); without the assertion Node would give an error similar to:

```
TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "xxxxxxx.json" needs an import assertion of type "json"
```

Ref: https://stackoverflow.com/questions/70106880/err-import-assertion-type-missing-for-import-of-json-file#comment126484258_70106896